idea_town/frontend/static-src/app/views/header-view.js

```diff
@@ -5,21 +5,66 @@ import template from '../templates/header-view';
 import scrollTemplate from '../templates/scroll-header-view';
 const changeHeaderOn = 100;
 
+import Model from 'ampersand-model';
+const HeaderModel = Model.extend({
```

Comment:
Another Ampersand fun fact: [ampersand-view](https://github.com/AmpersandJS/ampersand-view/blob/master/ampersand-view.js#L36) is a subclass of [ampersand-state](https://github.com/AmpersandJS/ampersand-state), also the parent class for [ampersand-model](https://github.com/ampersandjs/ampersand-model).

What that means is, you can declare all these props directly on the view itself and make bindings without the `model.` reference or the private model instance.